debian/rules

```diff
@@ -1,33 +1,70 @@
 #!/usr/bin/make -f
+#include /usr/share/dpkg/default.mk
```

Comment:
Why are you commenting the includes?

Reply:
I include /usr/share/dh-php/pkg-pecl.mk which already includes dpkg/default.mk

Reply:
Then please remove the commented includes if they are serving no purposes.